runhouse/servers/cluster_servlet.py

```diff
@@ -217,3 +244,139 @@ async def aclear_all_references_to_env_servlet_name(self, env_servlet_name: str)
         for key in deleted_keys:
             self._key_to_env_servlet_name.pop(key)
         return deleted_keys
+
+    ##############################################
+    # Cluster status functions
+    ##############################################
+
+    async def asend_status_info_to_den(self):
+        while True:
+            logger.info("Sending cluster status to Den")
+            interval_size = (await self.aget_cluster_config()).get(
+                "den_status_ping_interval"
+            )
+            try:
+                status: ResourceStatusData = await self.astatus()
+                status_data = {
+                    "status": "running",
+                    "resource_type": status.cluster_config.get("resource_type"),
+                    "data": dict(status),
+                }
+                cluster_uri = rns_client.format_rns_address(
+                    status.cluster_config.get("name")
+                )
+                api_server_url = status.cluster_config.get(
```

Comment:
it is a bit confusing to use `status.cluster_config` here, even though we already have. i'd use `(await self.aget_cluster_config())` just for clarity